script/BatchRegisterValidator.s.sol

```diff
@@ -120,4 +160,15 @@ contract BatchRegisterValidator is Script {
 
         return Permit({ deadline: deadline, amount: amount, v: v, r: r, s: s });
     }
+
+    function isJsonFile(string calldata str) external pure returns (bool) {
+        uint256 length = bytes(str).length;
+        if (length < 5) {
```

Comment:
Why length < 5 for file?  what are you checking here?

Reply:
ah it's the file extension `.json` . let's add this to the docs (make sure it's part of the commands to include .json